daml-lf/data/src/test/scala/com/digitalasset/daml/lf/data/RelationTest.scala

```diff
@@ -11,41 +11,44 @@ import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 
 final class RelationTest extends AnyPropSpec with Matchers with ScalaCheckPropertyChecks {
 
-  import Relation.{empty, flatten, invert, union}
-
   // an empty map and a map with exclusively empty values represent
   // the same relationship but the underlying structure is different
   private val nonEmptyRelations: Gen[Relation[Int, Char]] =
-    arbitrary[Relation[Int, Char]].suchThat(_.values.forall(_.nonEmpty))
```

Comment:
@S11001001 Have you any idea why I cannot use Relation[X, Y] here ? 

It is defined in the package object `com.daml.lf.data` as a alias for `Map[X, Set[Y]]`.

Reply:
Aliases can be a little funny, sometimes; it's probably trying to use this one with `C = Relation`:

https://github.com/typelevel/scalacheck/blob/1.13.5/src/main/scala/org/scalacheck/Arbitrary.scala#L347-L349

Reply:
Thanks